### PR TITLE
Add geojson support

### PR DESCRIPTION
**Pull request by xhacker**

### CODE REVIEW

File: ApplySyntax.sublime-settings [hunk 1]
```diff
@@ -181,7 +181,7 @@
         },
         {
             "syntax": "JavaScript/JSON",
-            "extensions": [".jshintrc", ".jscsrc", ".csslintrc", ".eslintrc", ".bowerrc"]
+            "extensions": [".jshintrc", ".jscsrc", ".csslintrc", ".eslintrc", ".bowerrc", "geojson"]
```
Comment (facelessuser) [inline ApplySyntax.sublime-settings]: Did you mean `.geojson` (with a dot)?  Just checking before I merge.

Comment (xhacker) [inline ApplySyntax.sublime-settings]: I believe `.geojson` only matches `.geojson`, while `geojson` matches `*.geojson`.

Comment (facelessuser) [inline ApplySyntax.sublime-settings]: Thanks, I was just making sure.  I don't know how geojson files normally look, so I just wanted to ask before I merge.

Comment (xhacker) [inline ApplySyntax.sublime-settings]: Thanks! 🙂

**Pull request merged**